ceph/ceph.py

```diff
@@ -1634,7 +1659,7 @@ def search_ethernet_interface(self, ceph_node_list):
             )
         )
         out, err = self.exec_command(cmd="sudo ls /sys/class/net | grep -v lo")
-        eth_interface_list = out.read().strip().decode().split("\n")
+        eth_interface_list = out.strip().split("\n")
```

Comment:
Since we are concatenating strings... this scenario would not work

Reply:
will check this as well.

Reply:
Did not find any issue.
> (Pdb) node.search_ethernet_interface(ceph_cluster.get_nodes())
> 2022-04-05 12:58:39,472 - ceph.ceph - INFO - Searching suitable ethernet interface on 10.0.210.165
> 2022-04-05 12:58:39,769 - ceph.ceph - INFO - Running command sudo ls /sys/class/net | grep -v lo on 10.0.210.165 timeout 600
> 2022-04-05 12:58:40,430 - ceph.ceph - INFO - Command completed successfully
> 2022-04-05 12:58:40,431 - ceph.ceph - INFO - Running command sudo ping -I eth0 -c 3 ceph-sunil03-9h69xs-node1-installer on 10.0.210.165 timeout 600
> 2022-04-05 12:58:43,537 - ceph.ceph - INFO - Command completed successfully
> 2022-04-05 12:58:43,538 - ceph.ceph - INFO - Running command sudo ping -I eth0 -c 3 ceph-sunil03-9h69xs-node2 on 10.0.210.165 timeout 600
> 2022-04-05 12:58:46,645 - ceph.ceph - INFO - Command completed successfully
> 2022-04-05 12:58:46,646 - ceph.ceph - INFO - Skipping ping check on localhost
> 2022-04-05 12:58:46,646 - ceph.ceph - INFO - Running command sudo ping -I eth0 -c 3 ceph-sunil03-9h69xs-node3 on 10.0.210.165 timeout 600
> 2022-04-05 12:58:49,821 - ceph.ceph - INFO - Command completed successfully
> 2022-04-05 12:58:49,822 - ceph.ceph - INFO - Suitable ethernet interface eth0 found on 10.0.211.232
> 'eth0'